### DISCUSSION TITLE
Add icons to home page ... CSS HTML etc for projects

### DISCUSSION THREAD

**Discussion by ItisNiGel** [community/community#22275] [Other Feature Feedback, Questions, & Ideas]
<p>Hi,</p>
<p>I am at a loss as to how to display other icons for projects on my home page … ie a project shows a red dot and  HTML but no CSS etc. and <em>visa versa</em> on other projects_!_</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/1X/36ef105f81dd635eab9828fb21933b9896e8c768.jpeg" data-download-href="/uploads/short-url/7PXYu5NbgoZNnPAjN1HBa3iLBY4.jpg?dl=1" title="Capture.JPG" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/10099150/181070016-56442c0a-6f96-418e-98ad-3e60e9b2519d.jpeg" alt="Capture.JPG" data-base62-sha1="7PXYu5NbgoZNnPAjN1HBa3iLBY4" width="659" height="500" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/1X/36ef105f81dd635eab9828fb21933b9896e8c768_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Capture.JPG</span><span class="informations">1029×780 71.6 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>Nigel :slight_smile:</p>

**Comment by AndreaGriffiths11** [comment, marked as answer]
<p>Hey <a class="mention" href="/iamnigelfrancis">@iamnigelfrancis</a>,</p>
<p>Thanks for being here! This process is done automatically by GitHub <a href="https://help.github.com/en/articles/about-repository-languages" rel="nofollow noopener">https://help.github.com/en/articles/about-repository-languages.</a></p>
<p>GitHub uses the open source <a href="https://github.com/github/linguist" rel="nofollow noopener">Linguist library</a> to determine file languages for syntax highlighting and repository statistics. Language statistics will update after you push changes to your default branch (which is usually <code>master</code>).</p>
<p>I hope this helps!</p>

**Comment by ItisNiGel** [comment]
<p>Hi,</p>
<p>Thank you. I thought that may be the case!</p>
<p>Kind Regards,</p>
<p>Nigel</p>
